osquery/utils/conversions/windows/windows_time.cpp

```diff
@@ -27,4 +29,38 @@ LONGLONG longIntToUnixtime(LARGE_INTEGER& li) {
   return ull.QuadPart / 10000000ULL - 11644473600ULL;
 }
 
+// Convert little endian Windows FILETIME to unix timestamp
+LONGLONG littleEndianToUnixTime(const std::string& time_data) {
+  // If timestamp is zero dont convert to UNIX Time
```

Comment:
Why not? I mean, I believe you, but the commend should reflect the why, and not just tell me what the `if` does.

Reply:
understood, i actually moved this _if_ claus out of the time conversion and into _shimcache_ and _userassist_
I expanded on the comment a little
Basically, sometimes the entries for  _shimcache_ or_userassist_ do not have a timestamp (instead its 0)
Let me know if the comment needs better clarification